internal/backup/csi_volumesnapshot.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
I'd like to clean up the file names to be a little clearer/consistent - so perhaps a standard of `<type>_action.go`, and this one becomes `volumesnapshot_action.go`?

Reply:
Yes. That's on my list for cleanup along with the names that the plugins are registered with.
Will put out that cleanup PR after the once in flight have landed.

Reply:
OK - can you just file an issue for visibility?

Reply:
this file is being newly added in this PR. so it makes sense to rename this now.